README.md

```diff
@@ -27,34 +27,27 @@ npm install --save react-native-view-more-text
 - **textStyle**([object, array]): Styles is passed to `Text` inside `ViewMoreText`
 (Refer to this [PR#8](https://github.com/nlt2390/react-native-view-more-text/pull/8))
 
-```javascript
+```jsx
+  ...redacted
```

Comment:
hi, is this a typo?

Reply:
Ah no, i wanted to say that the other part of the code was not needed as it is an example. 😓

Reply:
Done: 488d9c7a2ffcb88aeb76fbb6481f2c49694ff784.